Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -7,10 +7,12 @@ configuration:
   name: url
   required: true
   type: 0
-- display: API Token
-  name: token
-  required: true
-  type: 4
+- display: ''
```

Comment:
how this will look in previous versions?
In the previous versions the displaypassword will be hard coded `Password`.